scripts/gha/unity_installer.py

```diff
@@ -171,7 +171,7 @@ def install_unity(unity_version, platforms):
        "--verbose", unity_version,
        "-p", package_csv])
   # This will list what u3d has installed. For debugging purposes.
-  run([u3d, "list"])
+  # run([u3d, "list"]) TODO sort out the crash issue.
```

Comment:
I'd note that this crashes on Windows only, just to be clear

Reply:
I saw this crash on Windows too. I have a fix in another branch.

Reply:
I will leave this PR as is, and wait for Mou's fix.